elastic/apm-server
# Stop indexing "onboarding" documents

## Motivation/summary

Stop writing onboarding indices. From 8.0 on apm-server only uses data streams, and no privileges for creating indices should be required for the apm-server ES user.

## Checklist

~- [ ] Documentation has been updated~

## How to test these changes

1. Start apm-server in standalone mode
2. Check that no onboarding docs, indices, or templates are created

## Related issues

Closes #5994
